tests/js/open-behavior-tests.js

```diff
@@ -356,6 +356,70 @@ module.exports = {
     }
   },
 
+  testNewFile_downloadBeforeOpen_openLocalOnTimeOut_existing: async function () {
+    //This is a regression test.  If one were to logout before the openLocalRealm timeout
```

Comment:
A like to the issue would be nice

Reply:
I'll have to create one then.  Or should I link the help ticket directly?

Reply:
created https://github.com/realm/realm-js/issues/4453